com.unity.perception/Documentation~/Tutorial/Phase3.md

```diff
@@ -205,13 +205,13 @@ name        id        creation time         executions
                                               xBv3arj   Completed   2020-10-01 02:27:11    
 ```
 
-As seen above, each run has a name, an ID, a creation time, and a list of executions. Note that each "run" can have more than one "execution", as you can manually execute runs again using USim-CLI.
+As seen above, each run has a name, an ID, a creation time, and a list of executions. Note that each "run" can have more than one "execution", as you can manually execute runs again using USimCLI.
```

Comment:
Should this be renamed as well?

Reply:
USimCLI is the official name of the command-line interface, so I was hesitant to rename it in the tutorial.

Reply:
or maybe I'm wrong, it seems to only appear in the bundle itself